server/src/utils/fight/fightMethods.ts

```diff
@@ -981,10 +1000,13 @@ const activateSuper = (
         // Change pet owner
         pet.master = fighter.id;
         pet.team = fighter.team;
+        // Lower pet initiative
+        pet.initiative -= 1
       }
 
-      // Abort if no pet hypnotised
-      if (hypnotisedPets.length === 0) return false;
+      // Hypnotize opponent's brutes
+      const opponents = getOpponents({ fightData, fighter, bruteOnly: true })
+      opponents.forEach((opponent) => opponent.hypnotized = true);
```

Comment:
No need to assign them to a const if they are not used afterwards, just .forEach on the getOpponents directly